cmd/kyma/run/run.go

```diff
@@ -0,0 +1,19 @@
+package run
+
+import (
+	"github.com/kyma-project/cli/cmd/kyma/run/function"
+	"github.com/kyma-project/cli/internal/cli"
+	"github.com/spf13/cobra"
+)
+
+//NewCmd creates a new run command
+func NewCmd(o *cli.Options) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "run",
+		Short: "Runs resources locally.",
+		Long:  `Use this command to run resources from the local configuration.`,
```

Comment:
This might not be true for all subcommands. First level commands should have a very broad documentation since we will be adding many other subcommands in the future.

Reply:
@clebs This long description already seems quite generic - which part seems not to be true and how would you change it?

Reply:
The part that might not be true is `local configuration`.